### PR TITLE
add deploy operator

### PR DESCRIPTION
**Pull request by chxk**

### CODE REVIEW

File: deploy/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+      labels:
+        app: bagua-operator
+    spec:
+      containers:
+      - name: ares-operator
+        image: baguasys/operator:latest
+        imagePullPolicy: "IfNotPresent"
+        workingDir: /home/work
+        command:
+            - ./manager
```
Comment (eggiter) [inline deploy/deployment.yaml]: 添加liveness和readiness probe

File: deploy/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+kind: Deployment
+metadata:
+  namespace: bagua
+  name: bagua-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: bagua-operator
+  template:
```
Comment (eggiter) [inline deploy/deployment.yaml:14]: .spec.strategy:
    type: Recreate
    rollingUpdate: null

File: deploy/deployment.yaml (hunk 1 above)
Comment (chxk) [reply deploy/deployment.yaml]: 1

File: deploy/deployment.yaml (hunk 2 above)
Comment (chxk) [reply deploy/deployment.yaml:14]: 1

**Review by eggiter — APPROVED** [review]

**Pull request merged**